twitter-langchain/examples/chatbot/chatbot.ts

```diff
@@ -0,0 +1,215 @@
+import { TwitterToolkit } from "@coinbase/twitter-langchain";
+import { TwitterAgentkit } from "@coinbase/cdp-agentkit-core";
+import { HumanMessage } from "@langchain/core/messages";
+import { MemorySaver } from "@langchain/langgraph";
+import { createReactAgent } from "@langchain/langgraph/prebuilt";
+import { ChatOpenAI } from "@langchain/openai";
+import * as dotenv from "dotenv";
+import * as readline from "readline";
+
+dotenv.config();
+
+const modifier = `
+  You are a helpful agent that can interact with the Twitter (X) API using the Coinbase Developer Platform Twitter (X) Agentkit.
+  You are empowered to interact with Twitter (X) using your tools.
+
+  If someone asks you to do something you can't do with your currently available tools, you must say so, and encourage them to implement it themselves using the Twitter (X) API + Agentkit.
+  Recommend they go to https://developer.x.com/en/docs for more informaton.
+
+  Be concise and helpful with your responses.
+  Refrain from restating your tools' descriptions unless it is explicitly requested.
+`;
+
+/**
+ * Initialize the agent with Twitter (X) Agentkit
+ *
+ * @returns Agent executor and config
+ */
+async function initialize() {
+  // Initialize LLM
+  const llm = new ChatOpenAI({ model: "gpt-4o-mini" });
+
+  // Twitter (X) Agentkit options - these are loaded automatically via the ENV
+  const options = {
+    apiKey: "$TWITTER_API_KEY",
+    apiSecret: "$TWITTER_API_SECRET",
+    accessToken: "$TWITTER_ACCESS_TOKEN",
+    accessTokenSecret: "$TWITTER_ACCESS_TOKEN_SECRET",
+  };
+
+  // Twitter (X) Agentkit
+  const twitterAgentkit = new TwitterAgentkit(/*options*/);
```

Comment:
curious why the options param is commented out?

Reply:
just wanted to show it's an optional param if the user wants to override the ENV load for Twitter (X) app and token keys/secrets

Reply:
added a comment that should help clarify!

Reply:
thanks @stat. it's a nit but honestly I'd prefer removing the options var entirely in the Twitter `chatbot.ts` - like we do in the `cdp-langchain` chatbot when initializing `CdpAgentkit` - so as to discourage its usage and any accidents in people committing secrets in their code

Reply:
kk, roger that